remove computeBhpAtThpLimitProd() output from .DBG file

totto82 opened the pull request:
The current .DBG file is unreadable for cases with many wells due to hundreds of computeBhpAtThpLimitProd() output to the .DBG files. This PR removes them by default.

totto82 commented:
jenkins build this please

bska commented:
While I absolutely appreciate the desire to not output those messages by default, I'd rather prefer that we not make the option a compile-time constant.  Doing so means that re-enabling the messages is only available to those who compile the simulator from source.  Having a way to request additional diagnostics that does not require compiling the simulator is very valuable.

alfbr commented:
How about a command-line parameter to control verbosity of the DBG file, could possibly also be added to the PRT-file and terminal output. Both the DBG- and the PRT-file are reacing a point where the share amount of output makes them less useful.

OPMUSER commented:
I agree, which is why I have suggested a RPTDEBUG keyword, similar to the RPTSCHED keyword. So here perhaps:

```
RPTDEBUG
     WELLS=1 
/
```
@bska  and  @totto82  What do you think, or am I flogging a dead horse again :-)

OPMUSER commented:
I don't like the idea of yet more command line parameters. If we implement a RPTDEBUG keyword than it will give more flexibility on both the output type and timing. We can also use:

```
RPTDEBUG
  NONE
/ 
```
Switch the output off. This way we have some control so we can get debug at specific time intervals.

The horse came back to life for a brief moment :-)

totto82 commented:
> I'd rather prefer that we not make the option a compile-time constant.

My first suggestion was to removed the output completely, but then i hesitated and added a compile time constant. Are they useful at all at the current stage of development? Should we just removed them?

atgeirr commented:
I think adding the compile time option here is a good stopgap to make the DBG useful. Then if desired, we can make it a runtime option later with not too much fuzz. So I am inclined to merge this.

atgeirr commented:
Merging now, if we decide to add more runtime verbosity control it will not be hard to make these messages configurable as well.
